lib/oli_web/live/delivery/instructor_dashboard/helpers.ex

```diff
@@ -102,9 +100,44 @@ defmodule OliWeb.Delivery.InstructorDashboard.Helpers do
 
   def certificate_pending_approval_count(_students, _certificate), do: nil
 
-  defp return_page(graded_pages_and_section_resources, section, students) do
+  defp return_page(graded_pages_and_section_resources, section, _students) do
+    # Create a map of all section resource ids to their parent container labels
+    container_labels =
+      Oli.Delivery.Sections.SectionResourceDepot.containers(section.id)
```

Comment:
Create a map of section resource ids to their parent label containers.  No DB queries here, just a Depot call.